avocado/plugins/spawners/podman.py

```diff
@@ -142,7 +143,6 @@ def podman_version(self):
         if self._podman_version == (None, None, None):
             self._podman_version = self._get_podman_version()
         return self._podman_version
-        self._podman = None
```

Comment:
I can see that you removed it here. Can you please add this to the proper commit? Thank you.